app/models/miq_request.rb

```diff
@@ -34,11 +34,13 @@ class MiqRequest < ApplicationRecord
   virtual_column  :v_approved_by_email,  :type => :string,   :uses => {:miq_approvals => :stamper}
   virtual_column  :stamped_on,           :type => :datetime, :uses => :miq_approvals
   virtual_column  :v_allowed_tags,       :type => :string,   :uses => :workflow
+  virtual_column  :v_class,              :type => :string,   :uses => :workflow
   virtual_column  :request_type_display, :type => :string
   virtual_column  :resource_type,        :type => :string
   virtual_column  :state,                :type => :string
 
   delegate :allowed_tags,                :to => :workflow,   :prefix => :v
+  delegate :class,                       :to => :workflow,   :prefix => :v
```

Comment:
feels odd, too close to class, I know it has the v_ prefix, but I wonder if this should be named differently, maybe workflow_class ?